tests/sources/test_manual_sources.py

```diff
@@ -0,0 +1,47 @@
+from unittest import mock
+
+import pytest
+
+from telliot_feeds.sources.manual_sources.spot_price_input_source import SpotPriceManualSource
+from telliot_feeds.sources.manual_sources.string_query_manual_source import StringQueryManualSource
+from telliot_feeds.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_manual_spot_price_good_input(
+    monkeypatch,
+):
+    monkeypatch.setattr("builtins.input", lambda: "1234")
+    result, _ = await SpotPriceManualSource().fetch_new_datapoint()
+    assert result == 1234
+
+
+@pytest.mark.asyncio
+async def test_manual_spot_price_bad_input(capsys):
```

Comment:
I think we should have separate files for each manual source tests. They could get to be more than just one bad input, one good input test